openmessaging-api/src/main/java/io/openmessaging/api/order/OrderConsumer.java

```diff
@@ -27,7 +28,7 @@
  * @version OMS 1.2.0
  * @since OMS 1.2.0
  */
-public interface OrderConsumer extends Admin {
+public interface OrderConsumer extends ConsumerBase, Admin {
```

Comment:
Is "OrderedMessageConsumer" better describing the intention of this interface?

Reply:
@lizhanhui This will not be forward compatible, so it ’s better to keep this class name with OrderConsumer.

Reply:
@lizhanhui I also think "OrderedMessageConsumer" or "OrderlyMessageConsumer" is better, but considering the compatibility issue, it cannot be modified here

Reply:
OK. Let's keep it as it is for now. But to be honest, the name sounds weird, strange, and sort of cumbersome to get its interpretation.